values-en/strings.xml

```diff
@@ -82,7 +82,7 @@
             <string name="ms_end">Complete</string>
         <!--EXPLANATION-->
             <string name="explanation_permission_denied">In order to work correctly you have to grant Open GApps the permission to your external storage. Makes sense, doesn\'t it?</string>
-            <string name="explanation_adblock_detected">Adblocker detected!? Ads can be frustrating, but they keep the project and its developers alive! Please consider disabling your adblocker or a donation instead</string>
+            <string name="explanation_adblock_detected">Adblocker detected?! Ads can be frustrating, but they keep the project and its developers alive! Please consider disabling your adblocker or a donation instead</string>
```

Comment:
http://english.stackexchange.com/questions/531/is-there-a-standard-ordering-for-the-question-mark-and-the-exclamation-mark-used

There is no 'official' way, but according to the discussion there the !? is more popular in The Corpus of Contemporary American English